consul/resource_consul_service.go

```diff
@@ -81,7 +81,7 @@ func resourceConsulServiceCreate(d *schema.ResourceData, meta interface{}) error
 	// managed by the consul_node resource (or datasource)
 	nodeCheck, _, err := client.Catalog().Node(node, &consulapi.QueryOptions{Datacenter: dc})
 	if err != nil {
-		return fmt.Errorf("Cannot retrieve node: %v", err)
+		return fmt.Errorf("Cannot retrieve node '%s': %v", node, err)
```

Comment:
Tip for future: `%q` will correctly quote and escape strings.

Not worth changing, just thought I'd call it out in case you weren't aware.